src/workloads/docs/MonotonicSingleLoader.yml

```diff
@@ -1,7 +1,8 @@
 SchemaVersion: 2018-07-01
 Owner: "@mongodb/sharding"
 Description: |
-  TODO: TIG-3320
+  Loads a large set of documents in test.mycoll (Collection0 by default if mycoll is not specified)
```

Comment:
I'd probably keep it a bit more high level in case the behavior changes. Something like: "Loads a large set of documents with random value assigned to `field`" is probably good enough. The nitty gritty details can be left for the reader in the comments.